zerver/views/users.py

```diff
@@ -192,6 +195,10 @@ def patch_bot_backend(request, user_profile, email,
     if default_all_public_streams is not None:
         do_change_default_all_public_streams(bot, default_all_public_streams)
 
+    if payload_url is not None:
+        check_valid_interface_type(interface_type)
+        do_update_outgoing_webhook_service(bot, interface_type, payload_url)
```

Comment:
I'd rename this to `do_change_outgoing_webhook_service`, to be consistent with the other calls.